frontend/tsconfig.node.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "lib": ["ES2023"],
+    "module": "ESNext",
+    "skipLibCheck": true,
+
+    /* Bundler mode */
+    "moduleResolution": "bundler",
+    "allowImportingTsExtensions": true,
+    "isolatedModules": true,
+    "moduleDetection": "force",
+    "noEmit": true,
+
+    /* Linting */
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true
+  },
+  "include": ["vite.config.ts"]
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/2#issuecomment-2352110522>#115439</a><span> - 09/16/2024, 06:49 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The "include" array only contains "vite.config.ts", which may be too restrictive and exclude other necessary TypeScript files in the Node.js environment.
</div>
<div id="fix">
<b>Fix:</b> Expand the "include" array to cover all relevant TypeScript files in the Node.js environment, not just the Vite config file.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -18,5 +18,5 @@
      "noUnusedParameters": true,
      "noFallthroughCasesInSwitch": true
    },
 -  "include": ["vite.config.ts"]
 +  "include": ["vite.config.ts", "src/**/*.ts", "src/**/*.d.ts"]
  }
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged